subprojects/core/src/main/java/org/gradle/api/internal/tasks/properties/OutputFilePropertySpec.java

```diff
@@ -18,9 +18,11 @@
 
 import org.gradle.internal.file.TreeType;
 
+import javax.annotation.Nullable;
 import java.io.File;
 
 public interface OutputFilePropertySpec extends FilePropertySpec {
     TreeType getOutputType();
+    @Nullable
```

Comment:
Is the nullability which has been removed in https://github.com/gradle/gradle/pull/22594 now back?

Reply:
Yep, I ended up reverting it indeed. However, it is now actually used again.